File: in_go/src/pattern_7.go
package src

import "fmt"

func Pattern_7() {
	row := 5
	column := 9
	symbol := "*"

	for i := 0; i < row; i++ {

		for j := 0; j < column; j++ {
			if j <= row-1-i || j >= row-1+i {

				fmt.Print(symbol)
			} else {

				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_rust/src/pattern_7.rs
pub fn pattern_7() {
    let row = 5;
    let column = 9;
    let symbol = "*";

    for i in 0..row {
        for j in 0..column {
            if j <= row - 1 - i || j >= row - 1 + i {
                print!("{}", symbol);
            } else {
                print!(" ");
            }
        }
        println!();
    }
}
